src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -97,12 +98,28 @@ const getInstallationsWithAdmin = async (
 		const [isAdmin, numberOfRepos] = await Promise.all([checkAdmin, numberOfReposPromise]);
 		log.info("Number of repos in the org received via GraphQL: " + numberOfRepos);
 
+		let deferredInstallUrl: string | undefined;
+
+		// TODO: we should register the request in a separate POST endpoint. Short-cutting corners now for the spike,
+		// TODO: but must be addressed before rolling out for everyone, to avoid polluting redis
+		if (await booleanFlag(BooleanFlags.ENABLE_SUBSCRIPTION_DEFERRED_INSTALL, jiraHost) && !isAdmin) {
+			const payload: SubscriptionDeferredInstallPayload = {
+				installationIdPk,
+				gitHubInstallationId: installation.id,
+				orgName: installation.account.login,
+				gitHubServerAppIdPk: gitHubAppId
+			};
+			const requestId = await registerSubscriptionDeferredInstallPayloadRequest(payload);
+			deferredInstallUrl = `${envVars.APP_URL}/github/${gitHubAppUuid ? (gitHubAppUuid + "/") : ""}subscription-deferred-install/request/${requestId}`;
+		}
+
 		return {
 			...installation,
 			numberOfRepos,
 			isAdmin,
 			requiresSsoLogin: !!errors.find(err => err instanceof GithubClientSSOLoginError),
-			isIPBlocked: !!errors.find(err => err instanceof GithubClientBlockedIpError)
+			isIPBlocked: !!errors.find(err => err instanceof GithubClientBlockedIpError),
```

Comment:
/nit
while you are here, shouldn't it be
`isIPBlocked: errors.some(err=>err instanceof GithubClientBlockedIpError)`?

Reply:
My 2 cents on this: both achieve the same result but the latter is more readable and is a little clearer in terms of intent. errors.some conveys the purpose of checking if there exists any error of type GithubClientBlockedIpError and reads as "some errors satisfy the condition." 

At the end of the day, choice between them comes down to preference (I don't think we went this in-depth in coding conventions).